.github/workflows/main.yml

```diff
@@ -176,17 +176,20 @@ jobs:
       matrix:
         os: [ubuntu-latest, macos-12, windows-latest]
         python-version: ['3.8', '3.9', '3.10', '3.11', '3.12']
+        dist-type: ['whl', 'gz']
 
     steps:
       - name: Set up Python ${{ matrix.python-version }}
         uses: actions/setup-python@v5
         with:
           python-version: ${{ matrix.python-version }}
+
       - name: Install python dependencies
         run: |
           python -m pip install --user --upgrade pip
-          python -m pip install --upgrade wheel setuptools twine check-wheel-contents
```

Comment:
These packages aren't needed but were copy and pasted from the above step I'm guessing. We are still able to install and run the distributions without issue (verified in the test run I linked in the description)